### ISSUE TITLE
Make References/Links section consistent

### ISSUE THREAD

**Issue by seancdavis** [seancdavis/seancdavis-com#45]
This is just kind of written ad hoc today, but would be nice to formalize it in a consistent style.

**Comment by seancdavis** [comment]
What about making it part of the frontmatter, and then rendering it as part of the template. Maybe using a component, or maybe just shoving it directly into the post detail page for the time being?

**Comment by seancdavis** [comment]
Blocked by #44.

**Comment by seancdavis** [comment]
This is going to happen through introducing courses and tutorials, while leaving blog posts to just be loose and be what they are. So I'm going to mark this as `wontfix`.

**Comment by seancdavis** [comment]
I'm bringing this back as a follow-up to #131. I wanted originally to read posts on the fly and automatically include reference articles in the "More Reading" section at the bottom. But I think it would be good to instead have a resources section.

It would be nice to fetch (and cache) meta information for each of the references, but that's not necessary right out of the gate.
